src/Lemon/Containers/Converters/BinaryCia2NodeContainer.cs

```diff
@@ -75,6 +75,8 @@ public NodeContainerFormat Convert(BinaryFormat source)
             long contentOffset = (tmdOffset + header.TitleMetaLength).Pad(BlockSize);
             Node content = UnpackContent(title, stream, contentOffset);
 
+            title = title.TransformWith<Binary2TitleMetadata>();
```

Comment:
I prefer if we don't do this conversion here and we have every node by default in binary format. Also it's causing some tests to fail because the tests expects binary formats after a CIA. (See comment below for alternative).